DualCore/HelloWorld.csolution.yml

```diff
@@ -1,21 +1,25 @@
 solution:
-
-  created-for: CMSIS-Toolbox@2.5.0
+  created-for: CMSIS-Toolbox@2.6.0
   cdefault:
 
+  # List of tested compilers that can be selected
+  select-compiler:
+    - compiler: AC6
+    - compiler: GCC
+
   packs:
-    - pack: ARM::CMSIS
-    - pack: ARM::CMSIS-RTX
-    - pack: ARM::CMSIS-Compiler                # For retargeting stdout to UART      
-    - pack: NXP::K32L3A60_DFP@17.0.0
-    - pack: NXP::FRDM-K32L3A6_BSP@17.0.0 
+    - pack: ARM::CMSIS@^6.1.0
+    - pack: ARM::CMSIS-RTX@^5.9.0
+    - pack: ARM::CMSIS-Compiler@^2.1.0
+    - pack: NXP::K32L3A60_DFP@19.0.0
```

Comment:
Should we use @^19.0.0 with caret as well. Usually NXP releases major version increments where they are "free" to be incompatible but for minor or patch updates they are required to be compatible.

Reply:
Latest version uses @^ for device specific packs too.